Gemfile

```diff
@@ -20,6 +20,7 @@ gem 'devise_saml_authenticatable', '~> 1.2.2'
 gem 'foreman'
 gem 'haml'
 gem 'hyperresource'
+gem 'json', '>= 1.8.5'
```

Comment:
(there's a nasty bug wherein json 1.8.x where x < 5 will not compile on ruby 2.4.1, the current release version)